workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectDispatcherSystem.cs

```diff
@@ -2,13 +2,13 @@
 using System.Collections.Generic;
 using System.Linq;
 using Unity.Entities;
-using UnityEngine;
 
 namespace Improbable.Gdk.Core.GameObjectRepresentation
 {
     /// <summary>
     ///     Gathers incoming dispatcher ops and invokes callbacks on relevant GameObjects.
     /// </summary>
+    [DisableAutoCreation]
```

Comment:
Can do we a test that iterates over all Assembly types and ensures that they have the `DisableAutoCreation` set? Perhaps with a whitelist of systems that don't need it (if they exist)?

Reply:
I guess we could have a test that iterates only over core & feature modules and checks all the systems in there. right now we don't have any systems in there that don't need it